include/core/thermal_expansion_model.h

```diff
@@ -0,0 +1,113 @@
+/* ---------------------------------------------------------------------
+ *
+ * Copyright (C) 2019 -  by the Lethe authors
+ *
+ * This file is part of the Lethe library
+ *
+ * The Lethe library is free software; you can use it, redistribute
+ * it, and/or modify it under the terms of the GNU Lesser General
+ * Public License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * The full text of the license can be found in the file LICENSE at
+ * the top level of the Lethe distribution.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+#ifndef lethe_thermal_expansion_model_h
+#define lethe_thermal_expansion_model_h
+
+#include <core/physical_property_model.h>
+
+/**
+ * @brief ThermalExpansionModel. Abstract class that allows to calculate the
+ * thermal expansion coefficient (with unit of inverse temperature)
+ */
+class ThermalExpansionModel : public PhysicalPropertyModel
+{
+public:
+  /**
+   * @brief Instantiates and returns a pointer to a ThermalExpansionModel object by casting it to
+   * the proper child class
+   *
+   * @param physical_properties Parameters for a single fluid
+   */
+  static std::shared_ptr<ThermalExpansionModel>
+  model_cast(const Parameters::Fluid &fluid_properties);
+};
+
+
+/**
+ * @brief Constant thermal expansion coefficient.
+ */
+class ThermalExpansionConstant : public ThermalExpansionModel
+{
+public:
+  /**
+   * @brief Default constructor
+   */
+  ThermalExpansionConstant(const double p_thermal_expansion)
+    : thermal_expansion(p_thermal_expansion)
+  {}
+
+  /**
+   * @brief value Calculates the thermale expansion
+   * @param fields_value Value of the various field on which the property may depend.
+   * @return value of the physical property calculated with the fields_value
+   */
+  double
+  value(const std::map<field, double> & /*fields_value*/) override
```

Comment:
why `/*fields_value*/` is commented? Does this mean that right now nothing is parsed, but it should be the `fields_values` eventually? (so it's a WIP)

Reply:
interesting, I did not know this usage!